vng_api_common/oas.py

```diff
@@ -77,6 +77,11 @@ def obj_has_shape(obj: Union[list, dict], schema: dict, resource: str) -> bool:
         if value is None:
             if prop_schema.get("nullable", False):
                 continue
+            if (
```

Comment:
I'm actually not sure if this logic makes sense. Shouldn't the schema be fixed to mark it as nullable in those cases? What does the reference implementation do if it's empty/not required?

Reply:
The reason behind this change is to support nullable loose-fk fields (for example, `Besluit.zaak`).

In the reference implementation the correspondent fields are `URLField` with `blank=True`, so the value is not None, and this check is not applied. 
In `openzaak` in case of nullable `FkOrURLField` the value can be null, but the schema doesn't have `nullable` property. If it has a `nullable` property there will be schema difference between `openzaak` and the reference implementation